src/components/updatemenus/defaults.js

```diff
@@ -48,7 +50,9 @@ function menuDefaults(menuIn, menuOut, layoutOut) {
     var visible = coerce('visible', buttons.length > 0);
     if(!visible) return;
 
-    coerce('active');
+    // Default to zero since active must be *something*:
```

Comment:
Oops. The default is set. Removing this. I think this may have been a change that I thought was necessary while debugging but ultimately did not need.